### DISCUSSION TITLE
Is it possible to use OMR with RPi and two iPhones with different service providers (ATT and Verizon)

### DISCUSSION THREAD

**Discussion by vicen007** [Ysurac/openmptcprouter#2624] [Q&A] [answered]
I have recently discovered OMR, and it seems like it's a possible solution for a need of mine. I began doing some research to see if it is possible with my current hotspots provided by two iPhones. During my research, I came across this post: https://github.com/Ysurac/openmptcprouter/issues/88#issuecomment-418446483 and this quote indicates what I am trying to do is not possible:

> Note that you will not be able to use multiple iPhones or multiple Android phones over WiFi due to overlapping IPs (since neither allows you to change the hotspot IP address or DHCP pool).

Can someone please let me know if there is a workaround to this? Thank you!

**Comment by iambenmitchell** [comment, marked as answer]
I tried this a few months ago and it worked fine.

I used a: 

13 Pro Max - 4G
Xs max - 4G
iPhone 8 - 4G
EE broadband - ADSL

The iPhones were connected over USB though, perhaps Wi-Fi won't work.. but why would you want to use Wi-Fi anyway because you'd need a Wi-Fi card for each of the devices

**Reply by vicen007** [reply]
Ok thank you! What hardware did you use for the router? I was thinking of using a NanoPi R4S since RPi 4 are either out of stock or over priced!

**Reply by iambenmitchell** [reply]
> Ok thank you! What hardware did you use for the router? I was thinking of using a NanoPi R4S since RPi 4 are either out of stock or over priced!

I used a raspberry pi 4 as I was able to find one on eBay for MSRP. 

But if you just want to test it out first I'd recommend using an old laptop
